<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="checkBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>300</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>210</y>
      <width>121</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Start">
     <item>
      <widget class="QProgressBar" name="batteryLevelBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(164, 0, 0);
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="batteryLevelAdminSpinBox">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>30</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chargeBtn">
       <property name="text">
        <string>Charge</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="powerBtn">
       <property name="text">
        <string>Power On</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="powerCloseBtn">
       <property name="text">
        <string>Power Off</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="timeDuration">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>90</y>
      <width>120</width>
      <height>183</height>
     </rect>
    </property>
    <property name="title">
     <string>Time Duration</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="timeLeft">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(243, 243, 243);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dura_25min">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>25min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dura_45min">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>45min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dura_design">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>user design</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="session">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>90</y>
      <width>111</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Session</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="session_Delta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="session_Theta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Theta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="session_Alpha">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="session_Beta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Beta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="session_2">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>30</y>
      <width>61</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Intensity</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="intensity_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intensity_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="connection">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="connection_No">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connection_Okay">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connection_Excellent">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Excellent</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="userDuration">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>User Choice</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QRadioButton" name="user1_">
       <property name="text">
        <string>user1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="user2_">
       <property name="text">
        <string>user2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="user3_">
       <property name="text">
        <string>user3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="replayBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>241</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Replay</string>
    </property>
    <widget class="QLabel" name="reviewLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>241</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>111</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="pauseOrStop">
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>121</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="Record">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="View">
       <property name="text">
        <string>View Record</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
